bakerydemo/people/models.py

```diff
@@ -0,0 +1,162 @@
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.db import models
+from wagtail.admin.panels import FieldPanel
+from wagtail.api import APIField
+from wagtail.blocks import ChoiceBlock, StructBlock, URLBlock
+from wagtail.fields import StreamField
+from wagtail.models import Page
+from wagtail.search import index
+
+from bakerydemo.base.blocks import BaseStreamBlock
+
+from ..breads.models import Country
+
+
+class SocialMediaBlock(StructBlock):
+    """
+    Block for social media links
+    """
+
+    platform = ChoiceBlock(
+        choices=[
+            ("github", "GitHub"),
+            ("twitter", "Twitter/X"),
+            ("linkedin", "LinkedIn"),
+            ("instagram", "Instagram"),
+            ("facebook", "Facebook"),
+            ("mastodon", "Mastodon"),
+            ("website", "Personal Website"),
+        ],
+        help_text="Select the social media platform",
+    )
+    url = URLBlock(
+        help_text="Full URL to your profile (e.g., https://github.com/username)"
+    )
+
+    class Meta:
+        icon = "link"
+        label = "Social Media Link"
+
+
+class PersonPage(Page):
+    """
+    Detail view for a specific person
+    """
+
+    introduction = models.TextField(help_text="Text to describe the page", blank=True)
+    image = models.ForeignKey(
+        "wagtailimages.Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text="Landscape mode only; horizontal width between 1000px and 3000px.",
+    )
+    body = StreamField(
+        BaseStreamBlock(), verbose_name="Page body", blank=True, use_json_field=True
+    )
+
+    origin = models.ForeignKey(
```

Comment:
Noting this is a confusing term in the context of people, though it definitely makes sense to reuse `Country`. Maybe change the field label here at least?